### PR TITLE
CognitiveServices Personalizer 1.1 preview 1 (v0.2.0)

### PR DESCRIPTION
**Pull request by tjprescott**
In support of: https://github.com/Azure/sdk-release-request/issues/1547

### CODE REVIEW

File: sdk/cognitiveservices/azure-cognitiveservices-personalizer/azure/cognitiveservices/personalizer/models/__init__.py [hunk 1]
```diff
@@ -10,35 +10,106 @@
...
+    from ._models_py3 import PolicyResultTotalSummary
+    from ._models_py3 import RankableAction
+    from ._models_py3 import RankedAction
+    from ._models_py3 import RankRequest
+    from ._models_py3 import RankResponse
+    from ._models_py3 import RewardRequest
+    from ._models_py3 import ServiceConfiguration
+    from ._models_py3 import ServiceStatus
+    from ._models_py3 import SlotRequest
+    from ._models_py3 import SlotResponse
```
Comment (tjprescott) [inline sdk/cognitiveservices/azure-cognitiveservices-personalizer/azure/cognitiveservices/personalizer/models/__init__.py:40]: @lmazuel does this look right? I thought only V3 autorest had .models, but this was generated with `--verison=V2`

File: sdk/cognitiveservices/azure-cognitiveservices-personalizer/README.md [hunk 2]
```diff
@@ -1,34 +1,23 @@
...
 
-[Quickstart: Personalize client library for
-Python](https://docs.microsoft.com/azure/cognitive-services/personalizer/sdk-learning-loop?pivots=programming-language-python)
+For code examples, see [Personalizer](https://docs.microsoft.com/python/api/overview/azure/) on docs.microsoft.com.
 
-## Documentation
 
-For SDK reference documentation, see
-[Personalizer](https://docs.microsoft.com/python/api/overview/azure/) on
-docs.microsoft.com.
```
Comment (tjprescott) [inline sdk/cognitiveservices/azure-cognitiveservices-personalizer/README.md:26]: @weshaggard this commit was automated by the CI and it removes material that, presumably, should remain. Is there a way to disable this?

**Comment by tjprescott** [comment]
/azu run "python - cognitiveservices - ci"

File: sdk/cognitiveservices/azure-cognitiveservices-personalizer/README.md (hunk 2 above)
Comment (weshaggard) [reply sdk/cognitiveservices/azure-cognitiveservices-personalizer/README.md:26]: I'm not sure what automation generated this readme. @lmazuel is this being generated by autorest?

**Comment by tjprescott** [comment]
/azu run python - cognitiveservices - ci

File: sdk/cognitiveservices/azure-cognitiveservices-personalizer/README.md (hunk 2 above)
Comment (tjprescott) [reply sdk/cognitiveservices/azure-cognitiveservices-personalizer/README.md:26]: https://github.com/Azure/azure-sdk-for-python/pull/18720/commits/b1c50c4ae51f9d4c152d333cb75bd4aab5a4982a
This entire commit was made by automation, but I'm not sure which (or why for that matter).

File: sdk/cognitiveservices/azure-cognitiveservices-personalizer/azure/cognitiveservices/personalizer/version.py [hunk 3]
```diff
@@ -9,5 +9,5 @@
 # regenerated.
 # --------------------------------------------------------------------------
 
-VERSION = "0.1.0"
+VERSION = "0.2.0"
```
Comment (weshaggard) [inline sdk/cognitiveservices/azure-cognitiveservices-personalizer/azure/cognitiveservices/personalizer/version.py:12]: Is there a reason we are keeping these versions started with "0"? @lmazuel is this a pattern we expect to continue seeing or should this be changed to `1.0.0b1`?

**Pull request closed without merging**